server/playerServer/scripts/PlayerDatabase.js

```diff
@@ -350,7 +350,7 @@ PlayerDatabase.prototype.addProtectedRegion = function addProtectedRegion (useri
 				return;
 			}
 
-			this.database.query("INSERT INTO regions (owner, minX, minY, maxX, maxY, room) VALUES (?, ?, ?, ?, ?, ?)", [userid, minX, minY, maxX, maxY, room], function (err) {
+			this.database.query("INSERT INTO regions (owner, minX, minY, maxX, maxY, room, minRepAllowed) VALUES (?, ?, ?, ?, ?, ?, 300)", [userid, minX, minY, maxX, maxY, room], function (err) {
```

Comment:
Random magic number. Put it in a variable at the top of the file like var DEFAULT_MIN_REGION_REP = 300; and make it waaay higher. Like maybe even maxInt or close to it.